projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -886,6 +898,22 @@ function wpcom_launchpad_is_keep_building_enabled() {
 	return false;
 }
 
+/**
+ * Checks if the hosting flow task list is enabled.
+ *
+ * @return bool True if the task list is enabled, false otherwise.
+ */
+function wpcom_launchpad_is_hosting_flow_enabled() {
+	$intent  = get_option( 'site_intent', false );
+	$blog_id = get_current_blog_id();
+
+	if ( 'host-site' === $intent && $blog_id > 220443356 ) {
```

Comment:
Why do we need this `blog_id` checking?

Reply:
Not showing the new checklist for older sites/keeping the old checklist for those sites.

Reply:
I see. It would be good if we could make this less magical (e.g., introduce a helper function, or at least extract the variable somewhere.